Form/DataField/DataFieldType.php

```diff
@@ -183,13 +183,17 @@ public function getDefaultOptions($name)
 
     public function isDisabled($options)
     {
+        /** @var FieldType $fieldType */
+        $fieldType = $options['metadata'];
+
+        if (\in_array($fieldType->getName(), $options['disabled_fields'] ?? [], true)) {
+            return true;
+        }
+
         if (strcmp('cli', php_sapi_name()) === 0) {
             return false;
         }
 
```

Comment:
I would not disable it in CLI mode (perf and no meaning)
```suggestion
        if (strcmp('cli', php_sapi_name()) === 0) {
            return false;
        }

        /** @var FieldType $fieldType */
        $fieldType = $options['metadata'];

        if (\in_array($fieldType->getName(), $options['disabled_fields'] ?? [], true)) {
            return true;
        }

```

Reply:
fix in #344caf